examples/ecs/timers.rs

```diff
@@ -32,12 +32,12 @@ impl Default for Countdown {
 
 fn setup_system(commands: &mut Commands) {
     // Add an entity to the world with a timer
-    commands.spawn((Timer::from_seconds(5.0, false),));
+    commands.spawn((Timer::<Entity>::from_seconds(5.0, false),));
```

Comment:
I especially don't like using `Entity` as the associated type, I find it confusing